samples/bluetooth/central_gatt_write/README.rst

```diff
@@ -1,19 +1,20 @@
-.. _bluetooth_central_gatt_write:
+.. zephyr:code-sample:: ble_central_gatt_write
```

Comment:
Similar to the `BLE -> Bluetooth LE` change you did, we should probably do it for the references as well - Especially the ones that you've added/modified in this PR
```suggestion
.. zephyr:code-sample:: bluetooth_central_gatt_write
```

Reply:
They can also match the prefix we use in the stack as a shorter alternative:
```suggestion
.. zephyr:code-sample:: bt_le_central_gatt_write
```

Reply:
Sure, I can give it a try.

Reply:
I am actually not sure I have the knowledge required to pick the "right" prefix. I would frankly just name them "bt_<samplefoldername>". The point is really just to have *some kind* of unique ID for documentation purposes, it's not like it's directly exposed to the end user or anything

Reply:
> it's not like it's directly exposed to the end user or anything

Nor do we have any documentation or guidelines for naming schemes of sample labels :)

Doesn't need a fix in this PR, but arguably something that makes sense to make consistent later